### PR TITLE
[CI][VSTS] Clean .xip files.

### PR DESCRIPTION
**Pull request by mandel-macaque**
At some point either due to a manual error or a bug in provisionator a
*.xip file was left behind. Those files make provisioantor fail, for
example: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4361631&view=logs&j=f0137cdf-e292-5541-69aa-72303a08a0ba&t=59bea1c0-f907-5818-000e-f6fef9d3ffa2&l=230

We will remove those files to make sure provisionator works and we get
device tests back.

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]
yay

**Review by spouliot — APPROVED** [review]

**Review by mjbond-msft — APPROVED** [review]

File: tools/devops/automation/scripts/bash/clean-bot.sh [hunk 1]
```diff
@@ -105,6 +105,10 @@ oldXcodes=(
 	"/Applications/Xcode_12.2.0-rc.app"
 )
 
+# remove wrongly added .xip files under /Applications, confuses provisionator and 
+# are not needed and wrong
+sudo rm -Rf "/Applications/Xcode*.xip"
```
Comment (rolfbjarne) [inline tools/devops/automation/scripts/bash/clean-bot.sh:110]: The wildcard won't expand if you use quotes:

```
$ ls -lad "/Applications/Xcode*.app"
ls: /Applications/Xcode*.app: No such file or directory

$ ls -lad /Applications/Xcode*.app  
drwxr-xr-x  3 rolf  staff  96 Nov 11 05:00 /Applications/Xcode-12.3.0-beta1.app
lrwxr-xr-x  1 rolf  admin  30 Dec  9 16:37 /Applications/Xcode.app -> /Applications/Xcode_12.2.0.app
lrwxr-xr-x  1 rolf  admin  27 Sep 24  2019 /Applications/Xcode_10.2.1.app -> /Applications/Xcode1021.app
lrwxr-xr-x  1 rolf  admin  26 Jan 20  2020 /Applications/Xcode_11.3.0.app -> /Applications/Xcode113.app
drwxr-xr-x  3 rolf  staff  96 May  3  2020 /Applications/Xcode_11.5.0.app
drwxr-xr-x  3 rolf  staff  96 Jul  2  2020 /Applications/Xcode_11.6.0.app
lrwxr-xr-x  1 rolf  admin  30 Jul 23 11:44 /Applications/Xcode_11.6.app -> /Applications/Xcode_11.6.0.app
drwxr-xr-x  3 rolf  staff  96 Aug 12 19:30 /Applications/Xcode_11.7.0.app
drwxr-xr-x  3 rolf  staff  96 Sep  9 20:30 /Applications/Xcode_12.0.0.app
drwxr-xr-x  3 rolf  staff  96 Oct  8 10:25 /Applications/Xcode_12.1.0.app
drwxr-xr-x  3 rolf  staff  96 Oct 24 04:24 /Applications/Xcode_12.2.0.app
drwxr-xr-x  3 rolf  staff  96 Dec  3 23:20 /Applications/Xcode_12.3.0.app
```

```suggestion
sudo rm -Rf /Applications/Xcode*.xip
```

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

**Review by rolfbjarne — APPROVED** [review]

**Pull request merged**